static/assets/tutorials/cumulus/node-start-local-testing.sh

```diff
@@ -18,7 +18,7 @@ rm -rf /tmp/relay /tmp/parachain
 # Relay Validators with 2 validator chain spec
 ./target/release/polkadot --alice --validator --base-path /tmp/relay/alice --chain ../docs/static/assets/tutorials/cumulus/chain-specs/rococo-custom-2-raw.json --port 30333 --ws-port 9944
 
-./target/release/polkadot --bob --validator --base-path /tmp/relay/bob --chain ../docs/static/assets/tutorials/cumulus/chain-specs/rococo-custom-2-raw.json --port 30334 --ws-port 9945
+./target/release/polkadot --bob --validator --base-path /tmp/relay/bob --chain ../docs/static/assets/tutorials/cumulus/chain-specs/rococo-custom-2-raw.json --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/<bootnode_id_here> --port 30334 --ws-port 9945
```

Comment:
this was never needed before. Why is it now?

Reply:
you are correct, it auto-detects - not needed.